src/lib/components/ResponseThinkAndCheck/index.js

```diff
@@ -0,0 +1,58 @@
+import {html} from "lit-element";
+import {BaseResponseElement} from "../BaseResponseElement";
+import "./_styles.scss";
+
+/* eslint-disable require-jsdoc */
+class ResponseThinkAndCheck extends BaseResponseElement {
+  constructor() {
+    super();
+    this.prerenderedChildren = null;
+    this.option = null;
+
+    this.reset = this.reset.bind(this);
+  }
+
+  render() {
+    if (!this.prerenderedChildren) {
+      this.prerenderedChildren = [];
+      this.option = [];
+
+      for (const child of this.children) {
+        if (child.getAttribute("data-role") === "rationale") {
```

Comment:
Is there only ever going to be one of these?

Reply:
Right